### PR TITLE
Add Eslint to components

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Adding Eslint to components to improve code quality and add additional accessibility tests.

### CODE REVIEW

File: packages/web/scripts/copyCSSFile.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const fs = require('fs');
+import { copyFileSync } from 'fs';
```
Comment (ethanWallace) [inline packages/web/scripts/copyCSSFile.js:1]: I receive a ` Cannot use import statement outside a module` error when running `npm run build` with these changes.

**Review by ethanWallace — CHANGES_REQUESTED** [review]
It's looking good, thank you for adding this! Just one small thing I noticed.

File: packages/web/scripts/copyCSSFile.js (hunk 1 above)
Comment (melaniebmn) [reply packages/web/scripts/copyCSSFile.js]: Ah damn, that's too bad - I'll revert that change for now.

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Pull request merged**